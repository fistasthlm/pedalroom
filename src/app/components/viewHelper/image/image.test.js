import React from 'react';
import { shallow } from 'enzyme';
import Image from './image';

const setup = propOverrides => {
    const props = Object.assign({
        url: 'https//placekitten.com/500/500',
    }, propOverrides);

    const wrapper = shallow(<Image {...props} />);

    return {
        setup,
        wrapper,
    };
};

describe('Render', () => {
    it('should render component', () => {
        const { wrapper } = setup();

        expect(wrapper).toMatchSnapshot();
    });
});
