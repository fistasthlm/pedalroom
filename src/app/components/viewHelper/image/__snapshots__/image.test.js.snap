// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render component 1`] = `
<img
  alt=""
  src="https//placekitten.com/500/500"
/>
`;
